packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -60,25 +60,56 @@ const useIntentIconStyles = makeStyles({
   },
 });
 
+const useIntentIconStylesInverted = makeStyles({
+  success: {
+    color: tokens.colorPaletteGreenForegroundInverted,
+  },
+  error: {
+    color: tokens.colorPaletteRedForegroundInverted,
+  },
+  warning: {
+    color: tokens.colorPaletteYellowForegroundInverted,
+  },
+  info: {
+    color: tokens.colorNeutralForegroundInverted2,
+  },
+});
+
+const useActionButtonColorInverted = makeStyles({
+  action: {
+    color: tokens.colorBrandForegroundInverted,
+    '&:focus': { ...shorthands.outline('0') },
+    '&:focus-visible': {
+      zIndex: 1,
+      ...shorthands.outline('.2rem', 'solid', tokens.colorNeutralBackground5Pressed),
```

Comment:
Usually strokes and radii are defined with `px` and use tokens. Is there a reason you're not using tokens or `px` here?

Reply:
Been working in tmp too long. Good catch, thanks!